doc/client.md

```diff
@@ -1,33 +1,37 @@
+---
+title: Liquidityd
+---
+
+The liquidityd app for the liquidity pool module.
 
 # Liquidityd
 
 Implemented tx cli
 
-- [x]  `create-pool`   Create Liquidity pool with the specified pool-type, deposit-coins
-- [x]  `deposit`       Deposit submit to the batch of the Liquidity pool with the specified pool-id, deposit-coins
-- [x]  `swap`          Swap offer submit to the batch to the Liquidity pool with the specified pool-id with offer-coin, order-price, etc
-- [x]  `withdraw`      Withdraw submit to the batch from the Liquidity pool with the specified pool-id, pool-coin of the pool 
-
-Implemented query cli 
-
-- [x]    `batch`       Query details of a liquidity pool batch of the pool
-- [x]    `batches`     Query for all liquidity pools batch
-- [x]    `deposit`     Query for the deposit message on the batch of the liquidity pool specified pool-id and msg-index
-- [x]    `deposits`    Query for all deposit messages on the batch of the liquidity pool specified pool-id
-- [x]    `params`      Query the current liquidity parameters information
-- [x]    `pool`        Query details of a liquidity pool
-- [x]    `pools`       Query for all liquidity pools
-- [x]    `swap`        Query for the swap message on the batch of the liquidity pool specified pool-id and msg-index
-- [x]    `swaps`       Query for all swap messages on the batch of the liquidity pool specified pool-id
-- [x]    `withdraw`    Query for the withdraw message on the batch of the liquidity pool specified pool-id and msg-index
-- [x]    `withdraws`   Query for all withdraw messages on the batch of the liquidity pool specified pool-id
+- [x] `create-pool` Create liquidity pool
```

Comment:
I have been doing some studying about CLI command descriptions and I learned a lot! 
We want short descriptions that are verb based. We do not want to elaborate on the many parameters.
Good = Create liquidity pool

This content feels familiar to me, is this PR the right place to suggest CLI updates?  or can these updates by moved to the code? Please ping me for help so we can invest our time wisely and make Gravity DEX the most awesome tool in the universe

Reply:
I curated a fabulous list of CLI docs on this site https://www.writethedocs.org/guide/writing/style-guides/#command-line-resources

Reply:
@barriebyron I updated this `client.md` in [this commit](https://github.com/tendermint/liquidity/pull/201/commits/40db8bb200fbe68e760e105f5f31c02337c22de6). It would be great if you could take a look and provide any feedback. Then, we can resolve this conversation. Let's resolve conversation one by one and merge this PR as soon as we can.

Reply:
Yes! let's merge this PR and iterate for improvements